### ISSUE TITLE
Front end WEB UI

### ISSUE THREAD

**Issue by 0wwafa** [LostRuins/koboldcpp#1164]
Hello, I wish to add a feature to the front end HTML...
I don't find it in the repository...

I wish to add a print button, I already have the code to print the chat in a nice way.

**Comment by LostRuins** [comment]
The Lite repo is here https://github.com/LostRuins/lite.koboldai.net

**Comment by henk717** [comment]
Worth pointing out, KoboldAI Lite should have native printing functionality using the print of your browser.
Is this not working to your liking?

**Comment by 0wwafa** [comment]
> Worth pointing out, KoboldAI Lite should have native printing functionality using the print of your browser. Is this not working to your liking?

nope. because when printing to PDF it prints with white background (which is fine for real printing but not to have a nice pdf)
also, depending on the chat "theme" it does not print the full chat but only what's visible on the screen...
let's say there is a lot of space for improvement :D

**Comment by 0wwafa** [comment]
> The Lite repo is here https://github.com/LostRuins/lite.koboldai.net

oops.. I missed that :D
